### PR TITLE
added uk and intl home page routes

### PR DESCRIPTION
**Pull request by wheresrhys**

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -19,7 +19,9 @@ app.get('/__gtg', function(req, res) {
 	res.status(200).end();
 });
 
-app.get('/', require('./controllers/uk-front'));
+app.get('/intl', require('./controllers/front-page'));
```
Comment (commuterjoy) [inline server/app.js]: Can we change this to `international` or `world`?

File: server/app.js [hunk 2]
```diff
@@ -19,7 +19,9 @@ app.get('/__gtg', function(req, res) {
...
+app.get('/uk', require('./controllers/front-page'));
+app.get('/', require('./controllers/front-page'));
```
Comment (commuterjoy) [inline server/app.js]: I would 404 this route.

Comment (wheresrhys) [inline server/app.js]: will 404 once all the fastly redirects are working

File: server/app.js (hunk 1 above)
Comment (wheresrhys) [inline server/app.js]: will ask Mark, but stick to the existing convention for now (already added as route to service registry)

**Pull request merged**